novawallet/Modules/Staking/StakingMoreOptions/StakingMoreOptionsViewController.swift

```diff
@@ -0,0 +1,143 @@
+import UIKit
+import SoraFoundation
+
+final class StakingMoreOptionsViewController: UIViewController, ViewHolder {
+    typealias RootViewType = StakingMoreOptionsViewLayout
+
+    let presenter: StakingMoreOptionsPresenterProtocol
+    private var dAppModels: [DAppView.Model] = []
+    private var moreOptionsModels: [StakingDashboardDisabledViewModel] = []
+
+    init(
+        presenter: StakingMoreOptionsPresenterProtocol,
+        localizationManager: LocalizationManagerProtocol
+    ) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+        self.localizationManager = localizationManager
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        view = StakingMoreOptionsViewLayout()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupNavigationBarStyle()
+        setupCollectionView()
+        setupLocalization()
+        presenter.setup()
+    }
+
+    private func setupCollectionView() {
+        rootView.collectionView.registerCellClass(StakingMoreOptionCollectionViewCell.self)
+        rootView.collectionView.registerCellClass(DAppCollectionViewCell.self)
+        rootView.collectionView.registerClass(
+            TitleHeaderView.self,
+            forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader
+        )
+
+        rootView.collectionView.delegate = self
+        rootView.collectionView.dataSource = self
+    }
+
+    private func setupLocalization() {
+        title = R.string.localizable.multistakingMoreOptions(preferredLanguages: selectedLocale.rLanguages)
+        rootView.collectionView.reloadData()
+    }
+
+    private func setupNavigationBarStyle() {
+        guard let navigationBar = navigationController?.navigationBar else { return }
+
+        let statusBarHeight = UIApplication.shared.statusBarFrame.size.height
```

Comment:
I think I found a better way. Please, check ```StakingMainViewLayout```. But make sure you have merged the base branch first.